tests/Utils/Policies/TeamPolicy.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tests\Utils\Policies;
+
+use Illuminate\Auth\Access\Response;
+use Tests\Utils\Models\Team;
+
+final class TeamPolicy
+{
+    public function onlyTeams($user): bool
```

Comment:
type of `$user`?

Reply:
I intended the authorization to assert that the logged user type is `Team` and not null or other guard.